src/components/BookForm.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const BookForm = () => {
```

Comment:
- Move this component to `src/containers`
The convention about the container/component pattern is that components with some logic have to placed in the container folder